ui/apps/platform/src/Containers/ComplianceEnhanced/CompliancePage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PageSection, Title, Flex, FlexItem } from '@patternfly/react-core';
+
+function CompliancePage() {
+    return (
+        <>
```

Comment:
Not for this PR, can you follow up with Doron and Kylie whether there will be something like Vulnerabilities:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx#L25-L35

However, I strongly recommend blue info instead of yellow warning.

Reply:
Thinking ahead to plan for future pull requests, here are 2 questions:

Might this landing page consist of within-container routing
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx#L36-L54

If yes to previous, then keep this example in mind for child pages. `PageTitle` element is declarative way to specify page title for browser tab and history in go back/forward buttons.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPage.tsx#L61

Reply:
Both good callouts. I had noticed the same alert while submitting this PR – I'll bring up the question to UX. I suspect we will need within-container routing considering the scheduling scans part. I am picturing nested side nav similar to vulnerability management, one route being dashboard and the other being scheduled scans